### PR TITLE
Update 4-eslint-config.md

### PR DESCRIPTION
**Pull request by brookr**
Correct instructions for React 301 setup.

### CODE REVIEW

File: code-301/4-eslint-config.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-## ESLINT Configuration
+## ESLINT Configuration for React
 
-Copy the following line and paste into your terminal.  This will globally install the `eslint-plugin-react` package and will ensure that your React specific code is properly linted.
+Copy the following line and paste it into your terminal.  This will globally install the `eslint-plugin-react` package, and will ensure that your React-specific code is properly linted.
 
 `npm i -g eslint-plugin-react`
 
-Navigate to the root of your computer and add the following to your `eslintrc.json` file:
+Navigate to your home folder, `cd ~`, and add the following to `.eslintrc.json`, creating the file if it doesn't exist:
```
Comment (JacobKnaack) [inline code-301/4-eslint-config.md:9]: There is ambiguity here,  if the `.eslintrc.json` already exists, wouldn't students need to update each property if it's already defined?  Students use an eslint config in 201 that already has some of this defined right?

**Review by JacobKnaack — COMMENTED** [review]
can we update the eslintrc.json to include the things from 201 and add on the feeatures we want from React?:

```json
{  
    "extends": "eslint:recommended", 
    "plugins": [
        "react"
    ],
    "parserOptions": {
      "ecmaVersion": 6,
      "ecmaFeatures": {
            "impliedStrict": true,
            "jsx": true
            }
        },
        "env": {
             "browser": true,
             "es6": true 
        },
        "rules": {
            "react/jsx-uses-react": "error",
            "react/jsx-uses-vars": "error",
            "no-var": "error",
            "eqeqeq": ["error", "always"],
            "no-console": "off", 
            "no-undefined": "off",
            "indent": ["error", 2],
            "quotes": ["warn", "single"],
            "no-multi-spaces": [
                "warn",
                {
                    "exceptions": {
                        "VariableDeclarator": true
                        }
                    }
            ],
            "no-trailing-spaces": "warn",
            "new-cap": "warn",
            "no-redeclare": [
                "error",
                {
                    "builtinGlobals": true
                }
            ],
            "semi": [2, "always"],
            "eol-last": "error"
    }
}
```

File: code-301/4-eslint-config.md (hunk 1 above)
Comment (brookr) [reply code-301/4-eslint-config.md:9]: Great point, we should give them the entire file.

**Comment by brookr** [comment]
What is the source file that we give to students in 201 for their `.eslintrc.json`?

**Review by bnates — APPROVED** [review]

**Pull request merged**